src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Subscriber.kt

```diff
@@ -87,19 +90,16 @@ internal class TaskReceiver(tasksDirectory: String,
         this.tasksDir = directoryPath
     }
 
-    override fun processMessage(message: PubsubMessage) {
-        val request = CompilerFleet.CompilerFleetRequest.parseFrom(message.data)
-
-        val taskId = request.taskId
+    fun processCompileTask(taskId: String, rootFileName: String, zipBytes: ByteString): File {
```

Comment:
the name is kinda misleading: it does not process compile task, it just unzips

Reply:
Done